#
# I have not opened a case with NTAP support on this
# Certain times an empty $flag can make it through, which is noticed under use strict
#
--- lib/perl/NetApp/OCUMAPI-old.pm	2017-04-24 04:43:45.000000000 +0000
+++ lib/perl/NetApp/OCUMAPI.pm	2017-08-25 03:03:48.316206368 +0000
@@ -5460,7 +5460,7 @@
 
 sub IsArrayFlag {
     my ($self, $flag) = @_;
-    return ($flag & FIELD_ARRAY) == FIELD_ARRAY;
+    return $flag ? (($flag & FIELD_ARRAY) == FIELD_ARRAY) : 0;
 }
 
 sub IsEncryptedFlag {
